CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Installing / Getting Started
+
+## Prerequisites
+
+Install [Node.js, NPM](https://nodejs.org/en/download/) and [MongoDB](https://www.mongodb.com/download-center/community)
+
+## Setup Dev
+
+* Run.
```

Comment:
Also mention the steps to fork the repository and then clone it to the local machine here before running `npm install`.